packages/http-helper/src/cache/request_directive.ts

```diff
@@ -0,0 +1,38 @@
+//  @see
+//  - https://www.rfc-editor.org/rfc/rfc9111.html#name-request-directives
+//  - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#request_directives
+
+/**
+ *  - https://www.rfc-editor.org/rfc/rfc9111.html#name-max-age
+ */
+export const MAX_AGE = 'max-age' as const;
+
+/**
+ *  - https://www.rfc-editor.org/rfc/rfc9111.html#name-max-stale
+ */
+export const MAX_STALE = 'max-stale' as const;
+
+/**
+ *  - https://www.rfc-editor.org/rfc/rfc9111.html#name-min-fresh
+ */
+export const MIN_FRESH = 'min-fresh' as const;
+
+/**
+ *  - https://www.rfc-editor.org/rfc/rfc9111.html#name-no-cache
+ */
+export const NO_CACHE = 'no-cache' as const;
+
+/**
+ *  - https://www.rfc-editor.org/rfc/rfc9111.html#name-no-store
+ */
+export const NO_STORE = 'no-store' as const;
+
+/**
+ *  - https://www.rfc-editor.org/rfc/rfc9111.html#name-no-transform
+ */
+export const NO_TRANSFORM = 'no-transform' as const;
+
+/**
+ *  - https://www.rfc-editor.org/rfc/rfc9111.html#name-only-if-cached
+ */
+export const ONLY_IF_CACHED = 'only-if-cached' as const;
```

Comment:
According to the RFC and MDN, it seems `stale-if-error` can come to request.

https://datatracker.ietf.org/doc/html/rfc5861#section-4
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#cache_directives

Reply:
addressed:

- https://github.com/Nikkei/node-http-helper/pull/193/commits/0fcfa4be895848021d09a7d8c1e85d740d7fdff8
- https://github.com/Nikkei/node-http-helper/pull/193/commits/5679941e67d2be1cb550dd608383005e3a562756